stubs/redis/redis/typing.pyi

```diff
@@ -14,7 +14,7 @@ AbsExpiryT: TypeAlias = int | datetime
 ExpiryT: TypeAlias = float | timedelta
 ZScoreBoundT: TypeAlias = float | str
 BitfieldOffsetT: TypeAlias = int | str
-_StringLikeT: TypeAlias = bytes | str | memoryview
+_StringLikeT: TypeAlias = bytes | str | memoryview  # noqa: Y043
```

Comment:
Add a comment explaining why we ignore the error

Reply:
There's already one on line 9, but I'm happy to add another here if you like?

Reply:
Coolio